CMakeLists.txt

```diff
@@ -98,15 +99,17 @@ if (ENABLE_COVERAGE)
     COMMAND ${LCOV_EXE} --remove unfiltered_coverage.info -o coverage.info '*/external/*' '*/tests/*' '*/validation/*'
     WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
     COMMENT "Generating coverage report (coverage.info)")
-endif() 
+endif()
 
 # Support for valgrind (Linux only)
 include(add_memcheck_target)
 add_memcheck_target()
 
 # Testing
-include(CTest)
-enable_testing()
+if (ENABLE_TESTS OR ENABLE_SKYWALKER)
```

Comment:
So to disable testing you have to disable ENABLE_TESTS and disable ENABLE_SKYWALKER?